samples/web/CMakeLists.txt

```diff
@@ -19,29 +20,30 @@ if (CMAKE_CROSSCOMPILING)
     include(${IMPORT_EXECUTABLES})
 endif()
 
-set(MATC_TARGET mobile)
-set(MATC_FLAGS -a opengl)
+set(MATC_FLAGS -a opengl -m material -p mobile)
 if (CMAKE_BUILD_TYPE MATCHES Release)
     set(MATC_FLAGS -O ${MATC_FLAGS})
 endif()
 
 set(GENERATION_ROOT ${CMAKE_CURRENT_BINARY_DIR})
 file(MAKE_DIRECTORY "${GENERATION_ROOT}/generated/material/")
+file(MAKE_DIRECTORY "public/material/")
 
 set(MATERIAL_BINS)
 foreach (NAME ${MATERIAL_NAMES})
     set(mat_src "../materials/${NAME}.mat")
     get_filename_component(localname "${mat_src}" NAME_WE)
     get_filename_component(fullname "${mat_src}" ABSOLUTE)
-    set(output_path "${GENERATION_ROOT}/generated/material/${localname}.inc")
+    set(output_inc "${GENERATION_ROOT}/generated/material/${localname}.inc")
```

Comment:
Why do you output both .inc and .filamat?

Reply:
JS based web samples will use filamat, C++ based web samples use inc.